File: Ejemplos/Angular/Bierland/Bierland/Bierland.domain/Beer.cs
using System;
using System.Collections.Generic;

namespace Bierland.domain
{
    public class Beer
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int Qualification { get; set; }
        public virtual List<BeerPurchase> BeerPurchases { get; set; }
        public virtual List<BeerPubs> BeerPubs { get; set; }
        public bool IsDeleted { get; set; }
    }
}

File: Ejemplos/Angular/Bierland/bierland-web/src/Models/Beer.ts
export class Beer {
  id: number;
  name: string;
  description: string;
  qualification: number;

  constructor(id: number, name: string, description: string, qualification: number) {
    this.id = id;
    this.name = name;
    this.description = description;
    this.qualification = qualification;
  }
}
